code/game/objects/items/plushes.dm

```diff
@@ -203,8 +204,8 @@
 	else if(Kisser.partner == src && !plush_child)	//the one advancing does not take ownership of the child and we have a one child policy in the toyshop
 		user.visible_message("<span class='notice'>[user] is going to break [Kisser] and [src] by bashing them like that.</span>",
 									"<span class='notice'>[Kisser] passionately embraces [src] in your hands. Look away you perv!</span>")
-		plop(Kisser)
-		user.visible_message("<span class='notice'>Something drops at the feet of [user].</span>",
+		if(plop(Kisser))
+			user.visible_message("<span class='notice'>Something drops at the feet of [user].</span>",
 							"<span class='notice'>The miracle of oh god did that just come out of [src]?!</span>")
```

Comment:
even the code knows it is terrible

remove this entire feature

Reply:
People in glass houses shouldn't pick up the stone thrown in and cast it at another house, it might break another wall on its way out. Unless of course goofconomy left you an open air greenhouse?

Reply:
goofconomy is a vertical farm

Reply:
goofconomy is a house built by the invisible hand and therefore impenetrable